docs/contributing/accessibility/accessibility.mdx

```diff
@@ -0,0 +1,77 @@
+| :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [1.1 Text alternatives](https://www.w3.org/WAI/WCAG21/Understanding/text-alternatives)                                                                                                                                                                                                                                                                                                                                        | Include `alt-text` or aria-labels for any non-text elements that conveys information not provided elsewhere. If elements are not informative, they may be marked as `decorative`.                                                                                                                                                                                                                                                                                                |
+| [1.3.1 Info and relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships)<br/><br/> [1.3.2 Meaningful sequence](https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence)<br/><br/>[3.3.1 Error identification](https://www.w3.org/WAI/WCAG21/Understanding/error-identification)<br/><br/>[3.3.2 Labels or instructions](https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions) | Be sure content that relates is presented together and can be programmatically determined. <br/><br/>Use `aria-describedby` to associate content from one element with another. This is often used for form errors or helper text.                                                                                                                                                                                                                                               |
```

Comment:
> Be sure content that relates is presented together...

Would this be more clear to just say "Be sure related content is presented together..."

Reply:
Yep, that is better!